debezium-connector-mysql/pom.xml

```diff
@@ -390,6 +390,7 @@
               <mysql.port>4301</mysql.port>
               <mysql.gtid.port>4302</mysql.gtid.port>
               <mysql.gtid.replica.port>4303</mysql.gtid.replica.port>
+              <assembly.exclude.1>com.google.protobuf:protobuf-java:jar:*</assembly.exclude.1>
```

Comment:
Where is this one used?

Reply:
Mysql driver pulls it in

Reply:
@gunnarmorling It seems that protobuf is not necessary for JDBC-only usage so I am now excluding it at Maven dep level

Reply:
ok.